btf/i2cinit.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+sudo modprobe i2c-dev
+sudo modprobe i2c-stub chip_addr=0x24
```

Comment:
instead of hardcoding 0x24 we should have a variables file, and call i2cinit with an argument